types/react-dnd-multi-backend/index.d.ts

```diff
@@ -97,7 +98,10 @@ export interface PreviewProps {
  * This is frequently used with the Touch backend to provide a preview on mobile devices.
  */
 export class Preview extends PureComponent<PreviewProps> {}
-
+/**
+ * Pre-existing/default react-dnd-multi-backend transition available to use.
+  */
```

Comment:
This line has an extra space before the asterix, which is what is breaking the Travis build. If you remove the extra space it should build ok and be fine to merge.

Reply:
@dawnmist Thanks - missed that somehow :) Have fixed.